.gitignore

```diff
@@ -1,2 +1,3 @@
 # Ignore all the .txt files
-*.txt
\ No newline at end of file
+*.txt
+*node_modules
```

Comment:
- [ ] Kindly make the `test.md` file ignored by git so it must not be represented in the repo and it must be presented in `gitignore` file. :wink:

Reply:
@levy002 Updates have been made and submitted for re-review. Please check.